### DISCUSSION TITLE
Add color properties for Dev Toolbar App's UI Toolkit

### DISCUSSION THREAD

**Discussion by ktym4a** [withastro/roadmap#836] [Proposal]
# Summary

Provide build-in colors for better customization of UI Toolkit

# Background & Motivation

Currently only `astro-dev-toolbar-button` and `astro-dev-toolbar-badge` have style (color) properties.

https://docs.astro.build/en/reference/dev-toolbar-app-reference/#astro-dev-toolbar-button

https://docs.astro.build/en/reference/dev-toolbar-app-reference/#astro-dev-toolbar-badge

These also differ in the colors that can be set, but..,
For unify the color theme in the Dev Tool App, we should be able to set colors for other UI Tools.

# Goals

- Build-in colors can be set in all UI Tools

# Example

```ts
const toggle = document.createElement('astro-dev-toolbar-toggle');
toggle.toggleStyle = "red";

const highlight = document.createElement('astro-dev-toolbar-highlight');
highlight.highlightStyle = "yellow";
highlight.icon.bgStyle = "green";

const tooltip = document.createElement('astro-dev-toolbar-tooltip');
tooltip.tooltipStyle = "purple";
```

**Comment by Princesseuh** [comment]
I'm planning to do this in the coming weeks hopefully!

**Reply by ktym4a** [reply]
Glad to hear it because that is really what I want now! Thanks!
